File: flight_delay_prediction/v1/get/iOS SDK/FlightDelayPrediction.swift
import WAConvo

var waconvo: WAConvo = WAConvo(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

waconvo.travel.predictions.flightDelay.get(data: ["originLocationCode": "NCE",
    "destinationLocationCode": "IST",
    "departureDate": "2022-05-01",
    "departureTime": "18:20:00",
    "arrivalDate": "2022-05-01",
    "arrivalTime": "22:15:00",
    "aircraftCode": "321",
    "carrierCode": "TK",
    "flightNumber": "1816",
    "duration": "PT31H10M"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })

File: flight_delay_prediction/v1/get/Python SDK/flight_delay_prediction.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Will there be a delay from BRU to FRA? If so how much delay?
    '''
    response = waconvo.travel.predictions.flight_delay.get(originLocationCode='NCE', destinationLocationCode='IST',
                                                           departureDate='2022-08-01', departureTime='18:20:00',
                                                           arrivalDate='2022-08-01', arrivalTime='22:15:00',
                                                           aircraftCode='321', carrierCode='TK',
                                                           flightNumber='1816', duration='PT31H10M')
    print(response.data)
except ResponseError as error:
    raise error
